ci/setup_env.sh

```diff
@@ -1,6 +1,6 @@
 #!/usr/bin/env bash
 
-MAPI_SVC_ADDR=`kubectl get service -n openebs ci-openebs-maya-apiservice -o json | grep clusterIP | awk -F\" '{print $4}'`
+MAPI_SVC_ADDR=`kubectl get service -n openebs ci-openebs-apiservice -o json | grep clusterIP | awk -F\" '{print $4}'`
```

Comment:
I remember this was changed couple of days back. Can we write explicit comments on the names. Who decides these names? What is the source of truth?

Reply:
@amitd This is the fix cherry picked from kiran's fix and merge on 0.5 branch. https://github.com/openebs/maya/pull/303